docs/DeveloperGuide.md

```diff
@@ -529,6 +527,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | nurse                       | update the records of an elderly                              | the records are up to date without the need to delete and re-add the details                                                                     |
 | `* * *`  | nurse                       | add additional notes about an elderly                         | I can understand the elderly under my care better                                                                                                |
 | `* *`    | nurse                       | archive the details of elderlies                              | not clutter the system with the details of elderlies who have left the nursing home, but still keep the records of their stay for legal purposes |
+| `* * *`  | nurse                       | delete the next-of-kin information of an elderly              | easily update the next-of-kin information of an elderly easily, if the elderly's next-of-kin (hence point of contact) has changed                |
```

Comment:
The word 'easily' is used twice here? I feel that deleting Nok information is not really updating Nok information, updating would be done in `editElderly`? Perhaps we can say `easily delete next-of-kin information that are no longer relevant`?